rednotebook/files/rednotebook-highlight-style.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+ This file is part of GtkSourceView
+
+ Copyright (C) 2006-2007 GtkSourceView team
+ Author: Michael Monreal <michael.monreal@gmail.com>
```

Comment:
Could you please extend/update the copyright and authors information as you see fit in the two new XML files? Same for the `<author>` field below. Also, maybe we should add a comment explaining that the files come from GtkSourceView but were adapted for RedNotebook.